### DISCUSSION TITLE
Init a new project not working

### DISCUSSION THREAD

**Discussion by SunnyPirate** [quasarframework/quasar#10370] [Show and Tell]
with option as below 
1. sass with  scss syntax
2. ESLint && TypeScript && Vuex && Axios
Quasar CLI requires a minor change to the root component:
   /Users/qkb/Dev/test01/src/App.vue

  Please add: id="q-app" (or write #q-app if using Pug)
  to the outermost HTML element of the template.

Example:
  <template>
    <div id="q-app">
      ...
    </div>
  </template>

**Comment by ontwikkelfabriek** [comment]
@SunnyPirate

Please create a github Issue ( bug) if you think this is a...bug, in the issue section. This is the discussions forum. 

**------->    (Please don't create double posts in the forum)   <-----**
